politeiawww/cmd/politeiawwwcli/commands/logout.go

```diff
@@ -1,5 +1,14 @@
 package commands
 
+// Help message displayed for the command 'politeiawwwcli help logout'
+var LogoutCmdHelpMsg = `logout 
+
+Logout as a user or admin.
+
```

Comment:
Can we be explicit about the fact that there are no arguments?  So something like:
```
Arguments:
None

Results:
{}
```

Reply:
Implemented this